encode.go

```diff
@@ -56,11 +56,11 @@ func (e *MarshalerError) Error() string {
 //
 // Examples of struct field tags and their meanings:
 //
-//   // Field appears in INI as key "myName".
-//   Field int `ini:"myName"`
+//	// Field appears in INI as key "myName".
+//	Field int `ini:"myName"`
```

Comment:
This appears to be an IDE whitespace change that isn't directly related to this PR. I'd suggest dropping this change or moving it to a separate PR if you think 1 tab should be used instead of 3 spaces.

Reply:
If you split the change into two commits, one PR is fine.

Reply:
> If you split the change into two commits, one PR is fine.

Done.